packages/jelos/sources/scripts/set-audio

```diff
@@ -47,7 +54,7 @@ function set_audio_device() {
   then
     if [ "$(get_setting system.rg353v)" = "1" ]
     then
```

Comment:
Is it safe to directly overwrite asound.conf on the RG353V, or will this cause issues in other places?

Reply:
Yep, as long as you also expect to restart the emulator or ES.  May also need to bounce the volume service.